client/landing/gutenboarding/onboarding-block/style-preview/preview.tsx

```diff
@@ -128,7 +132,14 @@ const Preview: React.FunctionComponent< Props > = ( { fonts, viewport } ) => {
 					<div role="presentation" className="style-preview__preview-bar-dot" />
 				</div>
 			) }
-			<iframe ref={ iframe } className="style-preview__iframe" title="preview" />
+			<iframe
+				ref={ iframe }
+				className={ classNames( {
+					'style-preview__iframe': true,
+					hideScroll: viewport !== 'desktop',
+				} ) }
```

Comment:
```suggestion
				className={ classNames( 'style-preview__iframe', {
					hideScroll: viewport !== 'desktop',
				} ) }
```

also works :slightly_smiling_face: